### ISSUE TITLE
Squiz.PHP.DisallowMultipleAssignments.Found error when var assignment is on the same line as an open tag

### ISSUE THREAD

**Issue by westonruter** [squizlabs/PHP_CodeSniffer#1277]
The `Squiz.PHP.DisallowMultipleAssignments` sniff is giving a `Found` error “Assignments must be the first block of code on a line” for the following code snippet:

```php
<?php $edit_snapshot_text = __( 'Edit Snapshot', 'customize-snapshots' ); ?>
```

If I put a line break after `<?php` then it passes:

```php
<?php
$edit_snapshot_text = __( 'Edit Snapshot', 'customize-snapshots' );
?>
```

I would expect both snippets to not raise errors for `Squiz.PHP.DisallowMultipleAssignments`.

**Comment by gsherwood** [comment]
Fixed now. Thanks for the bug report.
